news/2020-02-26-new-featured-artist-sound-souler.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Sound Souler"
+date: 2020-02-26 09:00:00 +0000
+---
+
+We're proud to welcome **Sound Souler** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/70/header.jpg)
+
+Featuring on projects from *Team Arcstar* and *Milkyway TRAXX*, you might be already be familiar with **Sound Souler** from titles like *Muse Dash*, *Arcaea* and *OverRapid*. Suffice to say, he knows rhythm game music well, and we're excited to see what creations the osu! community will undertake with his sound. We've managed to snag **19** stonking tracks for your mapping experiments too, so there's plenty of things to choose from.
+
+The Mappers' Guild have been hard at work creating fancy preview sets showcasing the kind of magnificence you can get up to with his varied, electronica-styled sound, so make sure to check them out to get a feel for things. You can find them linked below, and I *highly* recommend checking them out.
+
+Here's a brief preview of what's available:
+
+### Aqua Stars
+
+Check out [this set from the Mappers' Guild](https://osu.ppy.sh/beatmapsets/1085144#osu/2269224) hosted by [Affirmation](https://osu.ppy.sh/users/6186628)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/70/previews/1916.mp3" type="audio/mpeg">
+</audio>
+
+### Absent Color
+
+Check out [this set from the Mappers' Guild](https://osu.ppy.sh/beatmapsets/1093465#osu/2285424) hosted by [Sonnyc](https://osu.ppy.sh/users/11771)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/70/previews/1914.mp3" type="audio/mpeg">
+</audio>
+
+### Gloom After Death
+
+Check out [this set](https://osu.ppy.sh/beatmapsets/1093465#osu/2285424) from [eiri-](https://osu.ppy.sh/users/3388410)!
```

Comment:
```suggestion
Check out [this set](https://osu.ppy.sh/beatmapsets/980331#osu/2051545) from [eiri-](https://osu.ppy.sh/users/3388410)!
```
The current URL links to Sonnyc's Absent Color set instead of eiri-'s Gloom After Death.